### PR TITLE
Fix refresh duplicated list

### PR DESCRIPTION
**Pull request by amilosmanli**
When refreshred, items will be fetched and pushed to the resource list again. This PR fixes that

### CODE REVIEW

**Comment by amilosmanli** [comment]
@benwinding I introduced a bug with #260. Appreciate if you can merge this small fix

File: src/providers/database/ResourceManager.ts
Comment (GwonHyeok) [inline src/providers/database/ResourceManager.ts:1]: How about changing it like this?

```typescript
resource.list = newDocs.map((d) => parseFireStoreDocument<IResourceItem>(d)); 
```

Comment (amilosmanli) [reply src/providers/database/ResourceManager.ts:1]: Hey @GwonHyeok. `forEach` here is not a JS array method. `getDocs` (line 94) returns a QuerySnapshot, which implements it's own `forEach` method. https://firebase.google.com/docs/reference/js/firestore_.querysnapshot.md#querysnapshotforeach

**Review by benwinding — APPROVED** [review]
LGTM 👌 Thanks for coming back with the fix 👍

**Comment by ucefkh** [comment]
I already fixed this last month in my version react-admin-firebase-01tek

but this is a workaround not a solution! this should be refetching every few secons! it should listen for changes!!

**Comment by black055** [comment]
This PR is merged but not successfully deployed, can you check that?

**Comment by mawnir** [comment]
The problem still exists

**Comment by benwinding** [comment]
Finally got CI working and deployed as `react-admin-firebase@4.1.3`

**Pull request merged**